rbd/rbd.go

```diff
@@ -225,20 +225,23 @@ func GetImageNames(ioctx *rados.IOContext) (names []string, err error) {
 	}
 }
 
-//
+// GetImage gets a reference to a previously created rbd image.
```

Comment:
I really hate this approach and would prefer that only `Create()` and `Open()` return an image... Wishful thinking, I guess, it will break most existing applications.

Reply:
I think this is in the leave it alone for now category.